Build opencv 3.3.0-rc and opencv_contrib 3.3.0-rc on travis-ci,but it's build error

I build opencv 3.3.0-rc and opencv_contrib 3.3.0-rc on travis-ci,but it's build error
my commond:
```bash
# Download OpenCV-contrib v3.3.0-rc
git clone https://github.com/opencv/opencv_contrib.git
cd opencv_contrib
git checkout 3.3.0-rc
cd ..

# Download OpenCV v3.3.0-rc
git clone https://github.com/opencv/opencv.git
cd opencv
git checkout 3.3.0-rc

mkdir build
cd build


cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D INSTALL_C_EXAMPLES=ON -D INSTALL_PYTHON_EXAMPLES=ON -D BUILD_EXAMPLES=ON -D WITH_QT=ON -D WITH_OPENGL=ON -D OPENCV_EXTRA_MODULES_PATH=../../opencv_contrib-3.3.0-rc/modules ..
# Run 'make' with four threads.
 make -j4

Install to OS.
sudo make install

```

error:
```
-- Found TBB: /usr/lib/libtbb.so
-- IPPICV: Download: ippicv_2017u2_lnx_intel64_20170418.tgz
CMake Error at cmake/OpenCVDownload.cmake:38 (file):
  Syntax error in cmake code at
    /home/travis/build/hihozhou/php-opencv/opencv/cmake/OpenCVDownload.cmake:38
  when parsing string
    # About to connect() to raw.githubusercontent.com port 443 (#0)
  # Trying 151.101.0.133...  connected
  # found 173 certificates in /etc/ssl/certs/ca-certificates.crt
  # 	 server certificate verification OK
  # 	 common name: www.github.com (matched)
  # 	 server certificate expiration date OK
  # 	 server certificate activation date OK
  # 	 certificate public key: RSA
  # 	 certificate version: #3
  # 	 subject: C=US,ST=California,L=San Francisco,O=GitHub\,
  Inc.,CN=www.github.com
  # 	 start date: Thu, 23 Mar 2017 00:00:00 GMT
  # 	 expire date: Wed, 13 May 2020 12:00:00 GMT
  # 	 issuer: C=US,O=DigiCert Inc,OU=www.digicert.com,CN=DigiCert SHA2 High
  Assurance Server CA
  # 	 compression: NULL
  # 	 cipher: AES-128-CBC
  # 	 MAC: SHA1
  # GET
  /opencv/opencv_3rdparty/a62e20676a60ee0ad6581e217fe7e4bada3b95db/ippicv/ippicv_2017u2_lnx_intel64_20170418.tgz
  HTTP/1.1
  # Host: raw.githubusercontent.com
  # Accept: */*
  # 
  # HTTP/1.1 200 OK
  # Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'
  # Strict-Transport-Security: max-age=31536000
  # X-Content-Type-Options: nosniff
  # X-Frame-Options: deny
  # X-XSS-Protection: 1; mode=block
  # ETag: "f7bc859fc42d18cc397b4bd50c32e23b8fc3037b"
  # Content-Type: application/octet-stream
  # Cache-Control: max-age=300
  # X-Geo-Block-List:
  # X-GitHub-Request-Id: 748A:4102:18C9566:1A27284:596EF69D
  # Content-Length: 40808979
  # Accept-Ranges: bytes
  # Date: Wed, 19 Jul 2017 06:05:18 GMT
  # Via: 1.1 varnish
  # Connection: keep-alive
  # X-Served-By: cache-ord1738-ORD
  # X-Cache: MISS
  # X-Cache-Hits: 0
  # X-Timer: S1500444318.536002,VS0,VE555
  # Vary: Authorization,Accept-Encoding
  # Access-Control-Allow-Origin: *
  # X-Fastly-Request-ID: 732282a4a0b03c90957420e9ee1a9e8984ccf3b8
  # Expires: Wed, 19 Jul 2017 06:10:18 GMT
  # 
  # 1�X�ippicv_2017u2_lnx_intel64_20170418.tgz
  \n
  Invalid escape sequence \,
Call Stack (most recent call first):
  cmake/OpenCVDownload.cmake:162 (ocv_download_log)
  3rdparty/ippicv/ippicv.cmake:42 (ocv_download)
  cmake/OpenCVFindIPP.cmake:243 (download_ippicv)
  cmake/OpenCVFindLibsPerf.cmake:12 (include)
  CMakeLists.txt:599 (include)
-- Configuring incomplete, errors occurred!
The command "cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local -D WITH_TBB=ON -D BUILD_NEW_PYTHON_SUPPORT=ON -D WITH_V4L=ON -D INSTALL_C_EXAMPLES=ON -D INSTALL_PYTHON_EXAMPLES=ON -D BUILD_EXAMPLES=ON -D WITH_QT=ON -D WITH_OPENGL=ON -D OPENCV_EXTRA_MODULES_PATH=../../opencv_contrib-3.3.0-rc/modules .." failed and exited with 1 during .
```

I found the garbled.

--- a/cmake/OpenCVDownload.cmake
+++ b/cmake/OpenCVDownload.cmake
@@ -156,10 +156,12 @@ function(ocv_download)
          TIMEOUT 600
          STATUS status
          LOG __log)
-    string(LENGTH "${__log}" __log_length)
-    if(__log_length LESS 65536)
-      string(REPLACE "\n" "\n# " __log "${__log}")
-      ocv_download_log("# ${__log}\n")
+    if(NOT OPENCV_SKIP_FILE_DOWNLOAD_DUMP)  # workaround problem with old CMake versions: "Invalid escape sequence"
+      string(LENGTH "${__log}" __log_length)
+      if(__log_length LESS 65536)
+        string(REPLACE "\n" "\n# " __log "${__log}")
+        ocv_download_log("# ${__log}\n")
+      endif()
     endif()
     if(NOT status EQUAL 0)
       set(msg_level FATAL_ERROR)

